### ISSUE TITLE
Issue with head/tail syntax in a tuple

### ISSUE THREAD

**Issue by jamonholmgren** [elixir-lang/elixir-tmbundle#76]
Example:

```elixir
  todo = "Learn Elixir"
  todos = []
  foo = {:bar, [todo | todos]} # syntax highlighting fails from here down
  {:noreply, [todo | todos]}
```

I'm not sure when I'll have time to take a crack at this, but I figured I'd file the issue anyway. I noticed this issue on Github and Sublime.

**Comment by endersstocker** [comment]
Fixed in #77

**Comment by jamonholmgren** [comment]
Thanks @endersstocker ! Really appreciate it.
